modules/promotions/src/getPromotions.ts

```
@@ -7,17 +7,21 @@ import { type Promotion, promotionSchema } from './schema';
 
 const dynamoClient = new DynamoDBClient(awsConfig);
 
-export const getPromotions = async (
-	stage: Stage,
-): Promise<Promotion[] | undefined> => {
+export const getPromotions = async (stage: Stage): Promise<Promotion[]> => {
 	const tableName = `MembershipSub-Promotions-${stage}`;
 	const input = {
 		TableName: tableName,
 	};
 	console.log(`Querying ${tableName}`);
 	const data = await dynamoClient.send(new ScanCommand(input));
 
-	return data.Items?.map((item) => {
+	if (data.Items === undefined) {
```

This slightly unrelated change is just to throw if we can't retrieve promotions from Dynamo rather than returning undefined. It didn't seem worth a separate PR!